### ISSUE TITLE
Please , use your own sprites , and make some decent sprites ffs 

### ISSUE THREAD

**Issue by Meltdown-Altair** [Iron-Miner/ProgressedContent#2]
Please , dont steal sprites from people's mods , and try to sprite your own stuff . Fix all your sprites , and follow the spriting guidelines ffs . It hurts my eyes just from seeing your bullcrap sprites .

**Comment by EmanuelG-Gaming** [comment]
> Please , dont steal sprites from people's mods , and try to sprite your own stuff . Fix all your sprites , and follow the spriting guidelines ffs . It hurts my eyes just from seeing your bullcrap sprites .

Yes. Just make your own sprites and make sure you never steal from other mods's sprites...

**Comment by amg2048** [comment]
> 
> 
> > Please , dont steal sprites from people's mods , and try to sprite your own stuff . Fix all your sprites , and follow the spriting guidelines ffs . It hurts my eyes just from seeing your bullcrap sprites .
> 
> Yes. Just make your own sprites and make sure you never steal from other mods's sprites...

exactly, just please make your own decent sprites
